Website/Program.cs

```diff
@@ -13,6 +13,8 @@
 builder.Services.AddScoped<IAuthTokenProvider, AuthTokenProvider>();
 builder.Services.AddScoped<AuthenticationStateProvider, AppAuthStateProvider>();
 builder.Services.AddScoped<IForumCategoryServices , ForumCategoryServices>();
+builder.Services.AddSingleton<SharpScape.Shared.Dto.Data>();

```

Comment:
Define a service with a descriptive name that lives in `SharpScape.Website.Services`

Reply:
Changed